data/deprecated.json

```
@@ -1205,6 +1205,126 @@
     "old": {"parking": "street"},
     "replace": {"parking": "lane"}
   },
+  {
+    "old": {"parking:lane:both": "yes"},
+    "replace": {"parking:both": "lane"}
```

```suggestion
    "old": {"parking:lane:both": "yes"},
    "replace": {"parking:both": "yes"}
```

When we build https://osmberlin.github.io/osm-tag-updater/#/manual , we made it that 
INPUT: `parking:lane:both=yes`
OUTPUT: `parking:both=yes`

Our idea being…
- it is likely `lane`, but we cannot be sure
- its better to have it a non standard `yes` so people get nudged to update it, that to auto update to something that might be wrong

---

But isn't it safe with parkin:**lane**:both that it is lane?